instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -286,8 +299,13 @@ def _request_ctx_ref() -> weakref.ReferenceType:
 
 
 def get_default_span_name():
+    method = sanitize_method(
+        flask.request.environ.get("REQUEST_METHOD", "").strip()
+    )
+    if method == "_OTHER":
+        method = "HTTP"
     try:
-        span_name = flask.request.url_rule.rule
+        span_name = method + " " + flask.request.url_rule.rule
```

Comment:
Maybe this just to be a bit neater

```suggestion
        span_name = f"{method} {flask.request.url_rule.rule}"
```